docs/user-guide/deploying-application/rollback-deployment.md

```diff
@@ -0,0 +1,36 @@
+# Rollback Deployment
+
+
+Deployments can be rolled back manually. After a deployment is completed, you can manually rollback a stage to a previously deployed image by retaining the same configuration or changing the configuration.
+
+**As an example**: You have deployed three different images V1, V2 and V3. And image V3 is currently deployed on your environment. Now, if you want to roll back the deployment from image V3 to image V1, you can retain the same configuration of the selected image V1 or change the configuration to `last saved config` or `last deployed config` of image V3.
+
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/rollback-deployment/rollback-icon.jpg)
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/rollback-deployment/select-config-to-deploy.png)
+
+1. Select `Rollback` in your deployed pipeline.
+
+2. On the `Rollback` page, select a configuration to deploy from the list:
+
+| Configurations | Description |
+| --- | --- |
+| **Last saved config** | Deploy the image with the latest saved configuration. |
+| **Last deployed config** | Deploy the image with the last deployed configuration. [As an example](#rollback-deployment): `The configuration of image V3`.|
+| **Config deployed with selected image** | Deploy the image with the configuration of a newly selected image. [As an example](#rollback-deployment): `The configuration of image V1`. |
```

Comment:
| **Config deployed with selected image** | Deploy the configs which were deployed with the selected image. [As an example](#rollback-deployment): `The configuration of image V1`. |